sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/TaskResourceMapper.java

```diff
@@ -230,9 +230,11 @@ private static OfferEvaluationStage toEvaluationStage(
             Optional<String> resourceId,
             Optional<String> persistenceId) {
         if (resourceSpec instanceof NamedVIPSpec) {
-            return new NamedVIPEvaluationStage((NamedVIPSpec) resourceSpec, taskSpecName, resourceId);
+            NamedVIPSpec namedVIPSpec = (NamedVIPSpec) resourceSpec;
```

Comment:
@mrbrowning Do you know why it got removed?

It seems reasonable to me for the `Spec`s to know how to generate their evaluation stages, but they are also in a very different package.